docs/Theming.md

```diff
@@ -405,6 +407,19 @@ const MyLayout = props => <Layout
 />;
 ```
 
+You can speciy the `Sidebar` size by setting the `size` property:
```

Comment:
Shoulnd't we document the `closedSize` prop too ?

Reply:
The current documentation for this type of theming is already verbose and unclear. We'll probably need to rewrite it somehow.

Besides, it's a really corner case IMO. We'll document it when we have an API doc like docz. Until then, let's consider it an easter egg.